### DISCUSSION TITLE
Question: Licensing and Fabulous project templates

### DISCUSSION THREAD

**Discussion by ChristophSchmidpeter** [fsprojects/Fabulous#1094] [General]
First of all, thanks for the amazing framework. This is the most promising UI framework I have seen.

## Question / Discussion

To my question:
The getting started guide recommends using the project templates e.g. via

`dotnet new fabulous-xf-app -n MyApplication`

However, in the generated file it includes a license header, and I don't quite understand what that implies when using the templates:

`// Copyright Fabulous contributors. See LICENSE.md for license.`

So what I understand is that the templates are licensed under the Apache 2.0 License, and thus I need to follow Apache 2.0 in my code-base if I use the templates. So for example, using the templates I need to comply to Apache 2.0 in the my own code base, or no? (E.g. keeping the license header in my code-base) Doesn't this limit the use-case scenarios of the templates?

**Comment by TimLariviere** [comment]
Hi @TitusLabienus. Thanks for your interest in Fabulous!

I'm no expert on the licensing subject but the way I understand it, Apache 2.0 only requires you to mention the copyright/license of Fabulous somewhere for your users to see. You don't need to have the license header in your own code files.
Also Apache 2.0 does not force you to use Apache 2.0 for your own code, you're free to choose your own license.

Usually, I've seen it done in the application About page. Like "Copyright of the app - Copyright lib1 - Copyright Fabulous - etc".

The license header of the templates is only here as a reminder to check the license of Fabulous if you intend to use it.
